packages/app-desktop/gui/dialogs.ts

```diff
@@ -21,14 +21,12 @@ class Dialogs {
 		}
 	}
 
-	public async prompt(message: string, title = '', defaultValue = '', options: any = null) {
-		if (options === null) options = {};
-
+	public async prompt(message: string, title = '', defaultValue = '', options: any = { cancel: true }) {
 		try {
 			const answer = await smalltalk.prompt(title, message, defaultValue, options);
 			return answer;
 		} catch (error) {
-			logger.error(error);
+			// dialog was cancelled
```

Comment:
It's annoying that cancelling and errors are handled in the same way, because there could actually be an error which will now be silenced. Would you mind changing this to a warning instead and add some info? Something like `logger.warn('Prompt appears to have been cancelled:', error)`